scripts/amd/gemm/tune_gemm.py

```diff
@@ -696,7 +811,9 @@ def main():
     # Check correctness from given configs
     if args.compare_wo_tuning:
         for (M, N, K, col_a, col_b, myConfig) in mnks:
-            test_correctness(M, N, K, col_a, col_b, dtype_a, dtype_b, dtype_c, init_type, myConfig, True)
+            if myConfig is None:
+                myConfig = get_default_config()
```

Comment:
Why would we want to compare the results without providing a tuning config?

Reply:
The script does not exclude the scenario that a user does not provide a tuning config and want to check the correctness of an input shape, so a better choice is to provide a default config instead of just crashed the program.  On the other hand, if a user provides a tuning config, it will use user's config.

Reply:
If we only have gemm size without any tuning config, it means we don't have an implementation of the gemm. Therefore, there is nothing to check, neither perf nor correctness.

Reply:
Changed to raising an exception if no config is provided.